sdk/include/opentelemetry/sdk/metrics/instrument.h

```diff
@@ -12,6 +12,10 @@
 #include "opentelemetry/sdk/metrics/record.h"
 #include "opentelemetry/version.h"
 
+#if defined(_MSC_VER)
+#pragma warning(disable:4250)
```

Comment:
Do you need pragma push-pop here?

Is this part of the headers that @lalitb is commenting out with a new feature flag?

Reply:
Added push-pop. Yes, this is part of the headers under the new feature flag.